style.css

```diff
@@ -0,0 +1,654 @@
+    opacity: 0; 
+
+}
+
+
+.balance div {
+    text-align: start;
+}
+
+.balance_label {
+    font-weight: 500;
+    font-size: larger;
+
+    margin-bottom: -0.8rem
+}
+
+.balance_data,
+.balance_data span {
+    font-size: small;
+    font-weight: 400;
+}
+
+.balnce_value {
+    font-size: 2.5rem;
+    margin-top: 1%;
+}
+
+.balance {
+    grid-column: 1/3;
+    display: flex;
+    justify-content: space-between;
+}
+
+
+.movements {
+    background:
+        linear-gradient(45deg, rgba(248, 112, 207, 0.834), rgb(250, 168, 223));
+    box-shadow: 0px 1px 5px rgb(249, 122, 251);
+    border-radius: 5px;
+
+    grid-row: 2/5;
+
+    display: flex;
+    flex-wrap: wrap;
+    align-content: flex-start;
+
+    overflow: scroll;
+}
+
+.movements__row {
+    background-color: white;
+    box-shadow: 0px 0px 3px 2px white;
+
+    height: 15%;
+    width: 100%;
+
+    margin: 1% 1% 0% 1%;
+
+    display: flex;
+    justify-content: space-around;
+
+    padding: 2%;
+}
+
+.movement-content {
+    display: flex;
+    justify-content: space-between;
+
+    width: 40%;
+}
+
+.movemnts_type,
+.move2 {
+    color: white;
+    border-radius: 10px;
+
+    font-size: large;
+    padding: 2%;
+}
+
+.movements_type--deposit {
+    background-color: rgba(6, 136, 13, 0.614);
+}
+
+.movements_type--withdrawal {
+    background-color: rgba(252, 0, 0, 0.858);
+}
+
+.movements__value {
+    font-size: 25px;
+}
+
+.movements__date {
+    font-size: 15px;
+    padding-left: 10px;
+}
+
+.operation {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    text-align: center;
+
+    border: 0.2px solid rgba(255, 255, 255, 0.46);
+    border-radius: 5px;
+
+    text-shadow: 2px 2px 3px white;
+
+    padding: 3%;
+
+
+}
+
+.operation h2 {
+    font-weight: 500;
+}
+
+.operation--loan {
+    box-shadow: 2px 2px 5px rgb(168, 247, 194);
+    background-color: rgb(168, 247, 194);
+    box-shadow: 1px 1px 3px 2px rgb(168, 247, 194),
+        1px 1px 4px 2px rgb(168, 247, 194),
+        0px 0px 2px 2px rgb(198, 248, 215),
+        1px 1px 4px 3px white;
+    grid-row: 3/4
+}
+
+.opration--transfer {
+    background: rgba(0, 255, 255, 0.339);
+    box-shadow: 0px 1px 5px rgb(0, 255, 255);
+    box-shadow: 1px 1px 3px 2px rgb(0, 255, 255),
+        1px 1px 4px 1px rgb(0, 255, 255),
+        0px 0px 2px 2px rgb(214, 242, 250),
+        1px 1px 4px 3px white,
+    ;
+    grid-row: 2/3;
+}
+
+.operation--close {
+    background: rgb(249, 176, 227);
+    box-shadow: 1px 1px 3px 2px rgb(251, 141, 218),
+        1px 1px 4px 1px rgb(255, 153, 224),
+        0px 0px 2px 2px rgb(249, 200, 234),
+        1px 1px 4px 3px white;
+    grid-row: 4/5
+}
+
+.summary {
+    grid-area: 5/1/6/3;
+    padding: 1%;
+
+    position: relative;
+}
+
+.p-content {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+
+    height: inherit;
+}
+
+.form__input {
+    background-color: rgba(255, 255, 255, 0.51);
+    border: none;
+    border-radius: 10px;
```

Comment:
border none but border radius has some value. Can you explain this @syedabdullahali